jscomps/PromptService.js

```diff
@@ -371,7 +423,8 @@ InternalPrompt.prototype = {
   },
 
   select: function select(aTitle, aText, aCount, aSelectList, aOutSelection) {
-    let p = this._getPrompt(aTitle, aText, [ PromptUtils.getLocaleString("OK") ]);
+    let p = this._getPrompt(aTitle, aText, [ "OK" ]);
+    p.setHint("select");
```

Comment:
Found it: https://github.com/sailfishos/sailfish-components-webview/pull/117#discussion_r666758477

I'd not introduce back blocking "embed:select" if not mandatory. With newer engine nsIPromptService.idl has asyncSelect as well.

Reply:
As you commented, this is only for hybrid app developers. Hence, can be added here for sake of completeness.